src/Raven.Client/Http/RequestExecutor.cs

```diff
@@ -735,18 +736,24 @@ protected virtual bool TryLoadFromCache(JsonOperationContext context)
                 }
                 catch (HttpRequestException e) // server down, network down
                 {
+                    Exception ex = e;
+                    if (e.InnerException is SocketException se && se.ErrorCode == 11001)
```

Comment:
Is that going to be the same in Linux? Is that the same in Windows?

Reply:
I would rather add the URL at all times always.

Reply:
It might also be different when SocketsHttpHandler is not used?